CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.12.0
+
+## Added
```

Comment:
We are missing:
- Added `check_extension_int` which allows checking the capabilities that return numbers instead of booleans: https://github.com/rust-vmm/kvm-ioctls/pull/202

Reply:
Added!